src/routes/index.js

```diff
@@ -1,21 +1,21 @@
-const express = require('express') ;
+import express from 'express';
 const app = express();
-const protection = require('../middlewares/check-auth');
+import protection from '../middlewares/check-auth';
 
-const { Router } = require('express');
+import { Router } from 'express';
+import { createRoom as _createRoom, getAllRooms, getRoomById, updateRoom as _updateRoom, deleteRoom as _deleteRoom, roomByHotel } from '../controllers/room';
 
 const router = Router();
 
-const controller = require('../controllers/room');
 
 //Rooms controllers
 
-const createRoom = controller.createRoom;
-const getRooms = controller.getAllRooms;
-const getRoom = controller.getRoomById;
-const updateRoom = controller.updateRoom;
-const deleteRoom = controller.deleteRoom;
-const getHotelRooms = controller.roomByHotel;
+const createRoom = _createRoom;
```

Comment:
You can destructure these lines (Line 13--18).

Reply:
I think there is nothing to destructuring